src/scripts.js

```diff
@@ -207,28 +206,40 @@ function openRecipeInfo(event) {
 function generateIngredients(recipe) {
   determineIfEnoughIngredients(recipe)
   return recipe.ingredients.map(i => {
-    return `${i.quantity.amount} ${i.quantity.unit} ${i.name}`
-  }).join("\n");
+    let quantity = domUpdates.formatQuantity(i.quantity.amount)
+    let unit = domUpdates.formatUnits(i.quantity.unit)
+
+    return `${quantity} ${unit} ${domUpdates.lowerCase(i.name)}`
+  }).join("\n")
 }
 
 function determineIfEnoughIngredients(selectedRecipe) {
   const shoppingList = []
-  const listItem = {}
 
   selectedRecipe.ingredients.forEach(recipeItem => {
     const userItem = user.pantry.find(item => item.ingredient === recipeItem.id)
+    const listItem = {}
 
     if (userItem) {
```

Comment:
we should look for a work around for this nested `if` statement, but I'm down to leave it if this was an uphill battle haha

Reply:
Yeah this function logic was a huge struggle and the nested if statement was the best we could come up with. Happy to come back to it in a later round of refactoring but think it's good for now.